pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,27 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.error("The app #{app.name} does not have a default external group.")
+        UI.error("Please make sure to pass group names to the `:groups` option.")
+        return
```

Comment:
@KrauseFx @ohayon Sorry, I actually managed to find this commit because I was searching form this in the code base. I don't agree that this should be an error because nothing actually fails.

We still get a build with an email. It could be a warning instead, and also how to remedy this without setting `:groups`.  For our case, we have two applications and one of them fails because for some reason, `app.default_external_group` returns nil, but not for the other app. We would like both to have external groups attached, but not sure how to remedy this either through fastlane or itunes connect. 

Just my two cents, hoping I can convince you to make this a warning/important and not an error as we're getting now in our CI even though it actually builds and submits. Thanks!